src/config/constants.ts

```diff
@@ -1,5 +1,7 @@
 import { StatusCodes } from 'http-status-codes';
 
+import { ThumbnailSize } from '@graasp/sdk/frontend';
```

Comment:
Actually I will need this in the front as well  👀  does that mean it should be moved out of frontend?

Reply:
Do you mean in the back-end as well? 😅